mega-service-agentQnA/docker-cleanup.sh

```
@@ -0,0 +1,82 @@
+case $image_cleanup in
+    2)
+        echo "Removing unused images..."
+        docker image prune -f
+        echo "✅ Unused images removed"
+        ;;
+    3)
+        echo "⚠️ WARNING: This will remove ALL Docker images ⚠️"
+        read -p "Are you sure? Type 'yes' to confirm: " confirmation
+        if [ "$confirmation" = "yes" ]; then
+            echo "Removing all images..."
+            docker rmi $(docker images -q) -f
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)